UnitySDK/Assets/ML-Agents/Scripts/Sensor/RenderTextureSensor.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using MLAgents.InferenceBrain;
+using UnityEngine;
+
+namespace MLAgents.Sensor
+{
+    class RenderTextureSensor : SensorBase
```

Comment:
This is almost identical to the CameraSensor, not sure if it's worth trying to de-dedupe some of the code.

Reply:
They use the same Utility under the hood, I think it is fine.

Reply:
Not quite, ObservationToTexture is overloaded :)